pyproject.toml

```diff
@@ -53,8 +53,19 @@ perovskite-solar-cell-database = { git = "https://github.com/FAIRmat-NFDI/nomad-
 extra-index-url = [
   "https://gitlab.mpcdf.mpg.de/api/v4/projects/2187/packages/pypi/simple",
 ]
-dev-dependencies = ["nomad-lab[parsing, infrastructure, dev]>=1.3.4"]
+dev-dependencies = [
+  "nomad-lab[parsing, infrastructure, dev]>=1.3.4",
+  "poethepoet>=0.29.0",
+]
 constraint-dependencies = ["hyperspy>=1.7.6"]
 
+[tool.poe.tasks]
```

Comment:
I understand that `poet` will automate setting up GUI env. Does it do every time we run `poe setup`? Ideally, we only need to set up the env once, right?

Reply:
Interestingly enough, we do this every time you run `setup_dev_env.sh` script, so I figured it doesn't harm here?